### ISSUE TITLE
TileLayer with 'noWrap:true' request extra tiles

### ISSUE THREAD

**Issue by songyumeng** [Leaflet/Leaflet#7181]
- [√ ] I've looked at the [documentation](http://leafletjs.com/reference.html) to make sure the behavior is documented and expected
- [√ ] I'm sure this is a Leaflet code issue, not an issue with my own code nor with the framework I'm using (Cordova, Ionic, Angular, React…)
- [√ ] I've searched through the issues to make sure it's not yet reported

**Steps to reproduce**
- new `L.TileLayer`
- set `noWrap:true` 
- open browser console，we can see 4 failed request 
![image](https://user-images.githubusercontent.com/29563916/85251953-46d56500-b48d-11ea-8ab3-180ce81ea71f.png)

**Expected behavior**
'noWrap:true' do not request extra tiles. When zoom = 1, only 4 tiles should be requested.

**Current behavior**
'noWrap:true' request extra tiles

**Environment**
- Leaflet version: 1.6.0
- Browser (with version): chrome 83.0.4103.97
- OS/Platform (with version):windows

**Minimal example reproducing the issue**

https://plnkr.co/edit/G2hSkBxwlLvnF7Nq

- [ √] this example is as simple as possible
- [ √] this example does not rely on any third party code

**Comment by IvanSanchez** [comment]
No, that's behaviour as expected: `noWrap` controls whether tile numbering wraps or not; it does not have an effect on how many tiles are loaded. It seems you're looking for https://leafletjs.com/reference-1.6.0.html#gridlayer-bounds .
